index.html

```diff
@@ -1,12 +1,232 @@
+                                    id="svg_5" />
+                                <path fill="#F6F7F7"
+                                    d="M11.840677626252173,4.694623074531556 c0.7190000000000004,-0.8710000000000004 1.204,-2.082 1.07,-3.287 c-1.035,0.04200000000000003 -2.2880000000000003,0.6900000000000007 -3.032,1.561 c-0.6660000000000007,0.77 -1.249,2.003 -1.091,3.184 C9.941677626252172,6.240623074531555 11.12267762625217,5.564623074531555 11.840677626252173,4.694623074531556 z"
+                                    id="svg_6" />
+                                <path fill="#BFBEBE"
+                                    d="M5.848677626252169,21.987623074531555 c1.3050000000000002,-0.05100000000000004 1.7970000000000002,-0.8430000000000004 3.374,-0.8430000000000004 c1.574,0 2.018,0.8430000000000004 3.396,0.8170000000000004 c1.401,-0.02500000000000002 2.292,-1.275 3.149,-2.529 c0.4350000000000002,-0.6340000000000005 0.7580000000000005,-1.26 0.981,-1.771 c-0.7580000000000005,-0.012000000000000007 -1.6030000000000002,-0.021000000000000015 -2.276,-0.021000000000000015 c-1.6540000000000008,0 -2.31,-0.3260000000000003 -5.421,-0.3260000000000003 c-3.115,0 -3.692,0.4180000000000002 -5.229,0.533 c-0.7050000000000004,0.05400000000000005 -1.466,0.05800000000000004 -2.084,0.04600000000000005 c0.266,0.546 0.557,1.048 0.8620000000000004,1.487 C3.4646776262521684,20.632623074531555 4.495677626252174,22.039623074531555 5.848677626252169,21.987623074531555 z"
+                                    id="svg_7" />
+                            </g>
+                        </g>
+                    </svg>
+                    Logo
+                </li>
+                <li>Store</li>
+                <li>Mac</li>
+                <li>iPod</li>
+                <li>iPhone</li>
+                <li>iPad</li>
+                <li>iTunes</li>
+                <li>Support</li>
+            </ul>
+            <div class="float-left buttom-decorator search-container container-style">
+                <form action="#">
+                    <div class="search-icon">
+
+                    </div>
+                    <input type="text" name="search">
+                </form>
+                <!-- <div >
+
+                </div> -->
```

Comment:
#### `REQUIRED`

Consider deleting commented out codes. If you need to review a previous version of your code, use git history.